packages/build-tools/src/utils/packageManager.ts

```diff
@@ -17,3 +20,12 @@ export function resolvePackageManager(directory: string): PackageManager {
 export function findPackagerRootDir(currentDir: string): string {
   return findYarnWorkspaceRoot(currentDir) ?? currentDir;
 }
+
+export async function readPackageJson(projectDir: string): Promise<any> {
+  const packageJsonPath = path.join(projectDir, 'package.json');
+  if (!(await fs.pathExists(packageJsonPath))) {
+    throw new Error(`package.json does not exist in ${projectDir}`);
+  }
+  const contents = await fs.readFile(packageJsonPath, 'utf-8');
+  return JSON.parse(contents);
+}
```

Comment:
Add a new line at the end of the file. (And fix your code editor settings.)

Reply:
The repo could use a lint command, or the right settings for tsc like some of the other repos, it doesn't warn, it doesn't fail the build.